### DISCUSSION TITLE
Fooocus username and password

### DISCUSSION THREAD

**Discussion by Neotutos** [lllyasviel/Fooocus#2264] [Q&A] [answered]
Hello, I have a problem, when I want to download fooocus it appears that I have to enter a username and password to be able to finish said download, what user and pass do I have to use, because I already tried with the github account and nothing
![Captura yaaa](https://github.com/lllyasviel/Fooocus/assets/159867967/aa4cdf90-0535-4ad8-a06b-616f95e24a81)

**Comment by mashb1t** [comment, marked as answer]
Which file are you trying to download?
Please use the official windows 7zip build https://github.com/lllyasviel/Fooocus/releases/download/release/Fooocus_win64_2-1-831.7z, where no password is required.

**Reply by Neotutos** [reply]
![hhha](https://github.com/lllyasviel/Fooocus/assets/159867967/e954633e-733a-40ea-8612-d921eff81fc6)
here, Is this file?

**Reply by mashb1t** [reply]
Yes.

**Reply by Neotutos** [reply]
Thank you very much, your link worked for me

**Comment by yalan2026** [comment]
thanx
